.ci/travis/travis.sh

```diff
@@ -105,6 +101,26 @@ javac8)
   done
   ;;
 
+javac9)
+  files=($(grep -Rl --include='*.java' ': Compilable with Java9' \
+        src/test/resources-noncompilable))
+  mkdir -p target
+  for file in "${files[@]}"
+  do
+    javac -d target "${file}"
```

Comment:
just for information: `-d` - https://docs.oracle.com/javase/9/tools/javac.htm#GUID-AEEC9F07-CB49-4E96-8BC7-BCC2C7F725C9__GUID-E31E2EF9-9FF9-43C1-B0D2-55B24E0421DC

Reply:
This is JDK-9 specific bug. For JDK-13 both `-d` and `--release` can be specified:
https://docs.oracle.com/en/java/javase/13/docs/specs/man/index.html

Reply:
Just checked:
```
/usr/lib/jvm/java-13/bin/javac --release 13 --enable-preview -d target src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocContentLocationOption.java 
```
works fine

Reply:
Please describe a bug, I just did a comment for next reviewer, as I was aware of "-d"

Reply:
> Please describe a bug, I just did a comment for next reviewer, as I was aware of "-d"

There is no bug, sorry for confusion. The note from your link, this one exactly:

> This option is not supported when using --release release to compile for JDK 9. 

wasn't for `-d` option. It was for `--boot-class-path`.